burnman/minerals/other.py

```diff
@@ -12,10 +12,34 @@
 
 from .. import mineral_helpers as helpers
 from ..mineral import Mineral
+from ..solidsolution import SolidSolution
+from ..solutionmodel import *
 
 from .SLB_2011 import periclase, wuestite, mg_perovskite, fe_perovskite
 
-
+class make_endmember(SolidSolution):
```

Comment:
> A final possibility is to write a function make_endmember that returns an instance of class MadeEndmember(Mineral).

I don't understand. Your class doesn't do anything SolidSolution doesn't already do. Why not return a ``SolidSolution``?